templates/home/api.html

```diff
@@ -21,121 +21,147 @@
 	<dd>Template <!-- Common design patter --></dd>
 
 	<dt>Last review</dt>
-	<dd>2019-05-09</dd>
+	<dd>2022-04-27</dd>
 
 	<dt>Conforming to</dt>
-	<dd>Going to be: Content and IA spec 2.0</dd>
+	<dd>Going to be: Content and IA spec 3.0</dd>
 
 	<dt>Specification</dt>
-	<dd><a href="https://www.canada.ca/en/treasury-board-secretariat/services/government-communications/canada-content-information-architecture-specification/templates-detailed-specifications/home-page.html">Home page for Canada.ca</a>
+	<dd><a href="https://design.canada.ca/mandatory-templates/home-page.html">Home page for Canada.ca</a>
 </dl>
 
 <h2>Technical specification</h2>
 
-<h3>Home page banner</h3>
+<h3>Your government</h3>
```

Comment:
Don't convert the component "home page banner" into "your government" Those are 2 different component, please create a new section for "your government".
The "Your government" component will be at version 2 not at version 3.

Reply:
Separated section are created for all 3 components: 

1. Home page banner;
2. Promotional banner;
3. Your Government